packages/interfaces/src/constants.ts

```diff
@@ -1,7 +1,7 @@
 /**
  * DefaultPubsubTopic is the default gossipsub topic to use for Waku.
  */
-export const DefaultPubsubTopic = "/waku/2/default-waku/proto";
+export const DefaultPubsubTopic = "/waku/2/rs/0/0";
```

Comment:
Did we deprecate the test fleet with one node that we had before for this pubsub topic?

Reply:
Do we have a test fleet specific for it? I wasn't aware
New nwaku versions don't support named sharding anymore, so if we have a test fleet in cluster 0 then it won't be able to upgrade to new versions. Although I'm not aware of us upgrading fleets that aren't on TWN or of Status.

So if it exists we have to upgrade its version and update it regularly after every release
cc @Ivansete-status

Reply:
Sounds good, also another thing to notice: we need to support `TWN` by default.

I'll pick up this PR and will continue in https://github.com/waku-org/js-waku/pull/2085

Reply:
Is supporting TWN by default part of the scope of this? I think it's just not allowing named sharding anymore but continuing in the same clusters as before. Supporting TWN from js side has the extra RLN component, which is in a separate scope I would guess?

Anyway, I think I'm close to finding and closing all the gaps which produce failed interop tests with latest master.
Turned this PR into draft and will add the fixes according to some changes we performed in nwaku which broke interop tests so they can be picked up or adapted in the PR you opened :))

Thanks so much!

Reply:
Updated it, now it is configured to TWN and tests are using the `0/0` pubsub.